plugins/GSdx/GSDialog.cpp

```diff
@@ -164,12 +164,27 @@ void GSDialog::SetTextAsInt(UINT id, int i)
 	SetText(id, buff);
 }
 
-void GSDialog::ComboBoxInit(UINT id, const vector<GSSetting>& settings, int32_t selectionValue, int32_t maxValue)
+void GSDialog::ComboBoxInit(UINT id, const vector<GSSetting> settings, int32_t selectionValue, int32_t maxValue)
 {
+	if (settings.empty()) return;
+
 	HWND hWnd = GetDlgItem(m_hWnd, id);
+	bool currentsel_found = false;
 
 	SendMessage(hWnd, CB_RESETCONTENT, 0, 0);
 
+	for (const GSSetting i : settings)
```

Comment:
Here some examples (because I played with the texture cache, I don't know if VS2013 support it).

```
plugins/GSdx/GSDevice.cpp-	auto compatible_texture = [&type, &format, &size, &msaa](const GSTexture* t) {
plugins/GSdx/GSDevice.cpp-		return (t->GetType() == type && t->GetFormat() == format && t->GetSize() == size && t->IsMSAA() == msaa);
plugins/GSdx/GSDevice.cpp-	};
plugins/GSdx/GSDevice.cpp-
plugins/GSdx/GSDevice.cpp:	const auto hit = std::find_if(m_pool.begin(), m_pool.end(), compatible_texture);
plugins/GSdx/GSDevice.cpp-
plugins/GSdx/GSDevice.cpp-	if (hit != m_pool.end()) {
plugins/GSdx/GSDevice.cpp-		GSTexture* t = *hit;
plugins/GSdx/GSDevice.cpp-		m_pool.erase(hit);
plugins/GSdx/GSDevice.cpp-		return t;
plugins/GSdx/GSDevice.cpp-	}
```
```
plugins/GSdx/GSTextureCache.cpp-	// Seach first in the same type cache
plugins/GSdx/GSTextureCache.cpp-	auto is_bp = [&bp](const Target* t) { return t->m_TEX0.TBP0 == bp; };
plugins/GSdx/GSTextureCache.cpp:	const auto hit = std::find_if(m_dst[type].begin(), m_dst[type].end(), is_bp);
plugins/GSdx/GSTextureCache.cpp-
plugins/GSdx/GSTextureCache.cpp-	if (hit != m_dst[type].end()) {
plugins/GSdx/GSTextureCache.cpp-		dst = *hit;
```